src/frontend/apps/markdown/MarkdownViewer/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect } from 'react';
+
+import { Box } from 'grommet';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { Loader } from 'components/Loader';
+import { getDecodedJwt } from 'data/appData';
+import { Nullable } from 'utils/types';
+
+import { MarkdownAppData } from 'apps/markdown/data/MarkdownAppData';
+import { getMarkdownDocumentTranslatedContent } from '../utils/translations';
+import LanguageSelector from '../LanguageSelector';
+
+const messages = defineMessages({
+  // Messages
+  translationNotFound: {
+    defaultMessage: 'Translation not found',
+    description: 'Error message displayed when translation is not found',
+    id: 'component.MarkdownEditor.translationNotFound',
+  },
+});
+
+const MarkdownViewer = () => {
+  const intl = useIntl();
+
+  const markdownDocument = MarkdownAppData.markdownDocument!;
```

Comment:
it seems like this field is required, so why do you type it as optional and nullable ?